datadog/dogstatsd/buffered_metrics.py

```diff
@@ -0,0 +1,63 @@
+import random
+from datadog.dogstatsd.metric_types import MetricType
+from datadog.dogstatsd.metrics import MetricAggregator
+
+
+class BufferedMetric(object):
+    def __init__(self, name, tags, metric_type, specified_rate=1.0, max_metric_samples=0):
```

Comment:
`max_metric_samples` appears to always be hardcoded to `0`. 
In the go client, I believe this is user configurable. Should we do the same in the python client?